src/plugin/controller.ts

```diff
@@ -7,7 +7,7 @@ import {
   checkType
 } from "./lintingFunctions";
 
-figma.showUI(__html__, { width: 360, height: 600 });
```

Comment:
Reducing the height because, on a 13'' inch mac book pro 600px takes almost all height.